src/System.IO.Compression/tests/ZipArchive/zip_InvalidParametersAndStrangeFiles.cs

```diff
@@ -326,55 +325,6 @@ public static async Task UnseekableVeryLargeArchive_DataDescriptor_Read_Zip64()
             }
```

Comment:
Why didn't you port the test that verifies you can read the file reported in https://github.com/dotnet/runtime/issues/1094?

Reply:
Got file not found exception, seems they don't point to same version of test data

Reply:
You can update the version to point to the new one.

https://github.com/dotnet/corefx/blob/8a3ffed558ddf943c1efa87d693227722d6af094/eng/Versions.props#L75

It doesn't seem right to not have a test for the change.

Reply:
Well the problem is test packaging and versioning is completely changed, that property is not exist in 5.0, there is new property `<SystemIOCompressionTestDataVersion>5.0.0-beta.20105.1</SystemIOCompressionTestDataVersion>` but not sure if copying that version directly would work, most likely not, I could try though CC @ViktorHofer

Reply:
That is actually worked locally, lets see if it broke in any CI leg

Reply:
Yes this should work as we still push to the feed that is used by 3.1. I just wanna warn that we currently don't use branching in dotnet/runtime-assets, so be cautious and keep in mind that updating the packages there has impact on servicing branches.

Reply:
As all CI legs passed assume no issue here